docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+
+services:
+  rookify:
+    image: "rookify-dev:latest"
+    container_name: rookify-dev
+    build:
+      context: .
+      dockerfile: DockerfileTesting
```

Comment:
Please use a customized entrypoint instead of an additional `Dockerfile` for that purpose.

See https://docs.docker.com/compose/compose-file/05-services/#entrypoint